packages/plugin-bezier/package.json

```diff
@@ -12,9 +12,7 @@
   },
   "bugs": "https://github.com/pmndrs/leva/issues",
   "peerDependencies": {
-    "leva": ">=0.9.23",
-    "react": ">=16.8.0",
-    "react-dom": ">=16.8.0"
```

Comment:
for the same reasons, wouldn't be better "listing" the ^major versions of leva this plugin works with: `"leva": "^0.9.23 || ^1.0.0"`?

NB: `|| ^1.0.0` is here to prepare the next One release

Reply:
I mean, a new leva release can break this version of the plugin, so in that case it would be nice having a peerDep ⚠️warning

Reply:
I'll do this at the end as yes, I'll introduce breaking changes everywhere 😬